lib/ain-ocean/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "ain-ocean"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
We cant use these. These needs to use the workspace versions or it'll result in many conflicts versions.